packages/security/src/components/Header/Header-story.js

```diff
@@ -22,12 +22,35 @@ import {
   profileWithAccountLongName,
 } from './_mocks_';
 
+import { InlineNotification, NotificationActionButton } from '../../';
+
 const headerProps = {
   labels,
   links,
 };
 
 disableCenteredStories(storiesOf(patterns('Header#legacy'), module))
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
```

Comment:
Looks like the Header is actually covering the inline notification for all the header stories.